tests/python/django/insecure_cookie/testdata/main.py

```diff
@@ -0,0 +1,8 @@
+def bad():
+  # bearer:expected python_django_insecure_cookie
+  HttpResponse.set_cookie("foo", "bar", max_age=None, secure=False, httponly=False)
```

Comment:
Should that be a static method, or an instance method? Examples that I looked at seemed to be instances.

If so, this could be helpful: https://github.com/Bearer/bearer-rules/pull/405